src/fu-util.c

```diff
@@ -4866,6 +4969,23 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Inhibit the system to prevent upgrades"),
 			      fu_util_inhibit);
+
+	fu_util_cmd_array_add(
+	    cmd_array,
+	    "security-harden",
+	    /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+	    _("[STREAM_ID]"),
+	    /* TRANSLATORS: command description */
+	    _("Automatically repair the system configuration to improve host security"),
+	    fu_util_auto_repair);
+
+	fu_util_cmd_array_add(cmd_array,
+			      "repair-list",
```

Comment:
This should be security-list I think.

Reply:
Is "security-harden-list" better since "security" lists the security attributes?